hooks/sentry_hook.py

```diff
@@ -104,25 +299,28 @@ def __init__(self, sentry_conn_id="sentry_dsn"):
         ignore_logger("airflow.jobs.backfill_job.BackfillJob")
         executor_name = conf.get("core", "EXECUTOR")
 
-        sentry_flask = FlaskIntegration()
-        integrations = [sentry_flask]
+        integrations = [FlaskIntegration(), SqlalchemyIntegration()]
 
         if executor_name == "CeleryExecutor":
             from sentry_sdk.integrations.celery import CeleryIntegration
 
-            sentry_celery = CeleryIntegration()
-            integrations += [sentry_celery]
+            integrations += [CeleryIntegration()]
 
         try:
             dsn = None
             conn = self.get_connection(sentry_conn_id)
             dsn = get_dsn(conn)
-            init(dsn=dsn, integrations=integrations)
+            sentry_sdk.init(dsn=dsn, integrations=integrations, traces_sample_rate=0.3)
```

Comment:
should we make this `0.3` magic number an environment variable with `0.3` the default maybe?